templates/registration/login.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html"%}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="POST">
+    {% csrf_token %}
+    <label>Username:
+      <input type="text" name="username">
+    </label><br>
+    <label>Password:
+      <input type="password" name="password">
+    </label><br>
+    <input type="submit" value="Submit">
+    <input type="hidden" name="next" value="{{ next }}">

```

Comment:
The `next` input field is correctly implemented to redirect users to their intended page after login, which is a good practice for user experience.